oraichain/chain.json

```diff
@@ -65,6 +66,19 @@
         "binaries": {
           "linux/amd64": "https://orai.s3.us-east-2.amazonaws.com/v0.41.3/oraid"
         }
+      },
+      {
+        "name": "v0.41.4",
```

Comment:
actually, since v0.41.4 is compatible with v0.41.3, we don't create a new version object in this case--we simply update the version object above